### DISCUSSION TITLE
Going back to swiss menu

### DISCUSSION THREAD

**Discussion by ibushigin** [emukidid/swiss-gc#567] [Q&A]
Hi there, 

I was wondering if it exists a shortcut to go back to swiss menu once a game was launched. Or do I need to restart the GC?

Cheers

**Comment by Extrems** [comment]
Look for In-Game Reset under Global Settings. Press Y for more information.

**Reply by ibushigin** [reply]
I will look for that

**Reply by ibushigin** [reply]
ty

**Reply by RexIsGod** [reply]
While using a 0.5r1026 Save exploit memory card to launch swiss, one can load the newest version of swiss (renaming it boot.dol) onto an sd2sp2 SD root directory to load the latest version of swiss.
One problem with this is: when you use the "In-Game Reset" along with the latest version of swiss (renamed to igr.dol, of course) installed onto the root of the SD card (sd2sp2), the "In-game reset" doesn't go back to the swiss menu, it restarts the Gamecube.  
In summary: It doesn't seem possible to reset to the latest swiss menu if the Save Exploit memory card swiss is of an earlier version.

**Reply by RexIsGod** [reply]
I'll reply to my own thread.  I've created a hacked save memory disc with the latest version of swiss.  I included the latest version of swiss on the SD2SP2 SD card (renaming it igr.dol).  It doesn't seem resetting into swiss from a game works anymore.  The gamecube soft restarts.  I've double checked the settings ensuring A+Z+Start is set to igr.dol in the swiss settings. At first I thought it was because I was using an older version from the memory card, but that's no longer the case.  
What am I doing wrong?

**Reply by Extrems** [reply]
What is the SD card formatted as? The reload stub doesn't support exFAT, and supporting it there isn't a high priority since all other applications don't support it either.

**Reply by RexIsGod** [reply]
It's fat32.  This function worked fine with swiss  0.5.  flawless.  Now that's i've updated to the latest swiss, it stopped working.

**Reply by RexIsGod** [reply]
After reviewing the card, it does appear to be exFAT.  What is the proper format to use?

**Reply by Extrems** [reply]
It must be either FAT12, FAT16, or FAT32.

**Reply by HappySpaceInvader** [reply]
> It must be either FAT12, FAT16, or FAT32.

Reformatted my card from exFat tp FAT32 and confirmed this fixed the issue.

Additionally, the reset combination is L + R + Z + B + Down

**Reply by Extrems** [reply]
The current combination is A + Z + Start. B + R + Z + Down is deprecated.
